app/src/main/res/values-pt-rBR/strings.xml

```diff
@@ -307,18 +306,25 @@
   <string name="displayseed_title">Frase de recuperação da carteira</string>
   <string name="displayseed_authenticate_button">Mostrar seed</string>
   <string name="displayseed_loading">Desbloqueando seed…</string>
-  <string name="displayseed_error">A seed não pôde ser lida. Tente novamente.\n\nMotivo: %1$s</string>
+  <string name="displayseed_error_generic">A seed não pôde ser lida. Tente novamente.</string>
+  <string name="displayseed_error_wrong_password">PIN incorreto.</string>
   <string name="displayseed_instructions"><![CDATA[
-    Esta lista de palavras, também conhecida como seed, é o backup da sua carteira. Salve as palavras em um local seguro (não neste telefone)!
+    Esta lista de palavras, também conhecida como seed, é o backup da sua carteira e permite o acesso total aos fundos contidos nela.
     <br /><br />
-    <b>Não compartilhe com ninguém. Qualquer pessoa que tenha acesso à sua lista de palavras pode roubar seus bitcoins.
+    <strong>Somente você possui esta seed. Mantenha-a privada.</strong>
     <br /><br />
-    <b>Não perca elas.</b> Se você perder sua lista de palavras e seu telefone, perderá seus bitcoins.
+    <strong>Não compartilhe esta seed com ninguém</strong>.
+    <br />
+    Cuidado com ataques phishing. Os desenvolvedores da Phoenix nunca pedirão sua seed.
+    <br /><br />
+    <strong>Não perca esta seed</strong>.
+    <br />
+    Salve-a em um local seguro (não neste telefone). Se você perder sua seed e seu telefone, você perderá seus fundos.
   ]]></string>
   <string name="displayseed_derivation_path">Seed BIP39 com caminho de derivação padrão BIP84</string>
   <string name="displayseed_words_td"><![CDATA[<td><font color="rgba(100,100,100,.3)"><small>#%1$d</small></font> <b>%2$s</b></td>]]></string>
-  <string name="displayseed_has_saved_seed_checkbox">Eu fiz o backup da minha seed</string>
-  <string name="displayseed_has_saved_seed_button">Confirmar</string>
+  <string name="displayseed_has_saved_seed_checkbox">Fiz o backup da minha seed, por exemplo, em um folha de papel.</string>
```

Comment:
```suggestion
  <string name="displayseed_has_saved_seed_checkbox">Fiz o backup da minha seed, por exemplo, em uma folha de papel.</string>
```

Reply:
Hi @dpad85, please approve this change. Thanks.

Reply:
Resolved with 1c12311